docs/Jira-Project-Migrations.md

```diff
@@ -0,0 +1,105 @@
+
+## Create Users/Groups/Roles In Target Jira Instance
+
+Jira's project import mechanism will create local users attached to the imported project if it can not find a matching username on the target Jira instance.  If the desire is to re-link the users to an external directory (i.e. Active Directory) and/or maintain user links to the issues, the users must be pre-created in the target Jira instance before the import and the username must match exactly.
+
+The same applies to groups and roles.  If the desire is to maintain the groups and roles as found on the imported project, these groups and roles need to be pre-created.
+
+## Initiate Jira Project Import
+
+After creating the empty project on the target Jira instance and setting all the necessary configurations to match the source project, the project import can be attempted.
+
+### Project Import Outline
+
+*  [Copy Backup Files From Source To Destination](#copy-backup-files-from-source-to-destination)
+*  [Initiate Jira Project Import](#initiate-jira-project-import)
+*  [Project Import Notes](#project-import-notes)
+*  [Project Import Validation](#project-import-validation)
+
+### Copy Backup Files From Source To Destination
+
+Projects are imported using the full system backup files of a Jira deployment.  To initiate the project import, copy the full backup files of the source Jira instance created previously to the target Jira instance.  Copy the backup database `zip` file and the associated project attachment folder to `/var/atlassian/application-data/jira/import`.  Ensure the user and group ownership is `jira`.  Otherwise, the project import will fail.
+
+### Initiate Jira Project Import
+
+After the requisite files are copied to the appropriate location, the project import can be initiate by the following steps:
+
+*  Log into the web management console using an account with Jira administrator privileges.
+*  Click the gear wheel icon at the top right and select `System` under `JIRA ADMINISTRATION`.
+*  In the Administration page, select `Project import` under `IMPORT AND EXPORT`.
+*  Type in the file name of the backup `zip` file copied over in a previous step and click `Next`.  This will initiate a scan of the backup files and provide a list of available projects to import.
+*  Select the desired project to import from the drop-down menu and ensure there are no warnings or issues raised.  Check the option `Overwrite Project Details`. Click `Next` to proceed.
+*  Review the validation results and resolve any items that have not passed as indicated by a yellow exclamation point.  Items that passed are indicated by a green check-mark.
+*  Once all items are resolved, the import is ready to proceed.  Click `Import` to proceed.
+
+Depending on the size of the project, the import process can be a lengthy process so be patient.  Confirm the import is progressing by checking the status/completion bar.
```

Comment:
Any recommendations on (presumably temporarily) moving the target to a larger instance-type (or other reconfiguration options) to accelerate the process

Reply:
I'm thinking this maybe should go into the pre-migration notes.  The target instance and DB may need to be upgraded to larger instances permanently if the imported project adds a large number of users and issues.

Reply:
Good thought; good catch.